src/client/main.c

```diff
@@ -42,27 +43,28 @@ int method_version(UNUSED Command *command, UNUSED void *userdata) {
 }
 
 const Method methods[] = {
-        { "help",            0, 0,       OPT_NONE,                                method_help,            usage_bluechi                },
-        { "list-unit-files", 0, 1,       OPT_FILTER,                              method_list_unit_files, usage_method_list_unit_files },
-        { "is-enabled",      2, 2,       OPT_NONE,                                method_is_enabled,      usage_method_is_enabled      },
```

Comment:
You have removed `is-enabled` method, this is why integration test is failing

Reply:
Changed

Reply:
@mwperina Artiom has re-added it, before that `is-enabled` wasn't there and the corresponding test failed because of that

Reply:
I think there was some rebase issue

Reply:
And it is still there! Looking at the commits, @ArtiomDivak you are adding `is-enabled` method, although it should be already there. Please do a proper rebase.

Reply:
I recently rebased and added the line. Now the issue is gone. Not sure what made it work

Reply:
Sorry, you have removed it in the first commit and added it in the second commit.

Reply:
Please just undo the removal in this commit https://github.com/eclipse-bluechi/bluechi/pull/951/commits/3e554d2db0dad610028823001df5212f49be38f7 rather than removing it and adding it in the next commit.

Or even better, just squash these two commits to one. You are just creating a commit that will fail if checked out. No need for this complication, work with a single commit per PR, it is good enough. If you decide to create separate commits, you need to make sure that each commit is independent and the system will run correctly if built on top of each one of them.